src/js/store/actions-list.js

```diff
@@ -1,4 +0,0 @@
-export const START_RECORDING = 'START_RECORDING';
-export const ON_VOICE_RECORDED = 'ON_VOICE_RECORDED';
-export const SEARCH_IN_PROGRESS = 'SEARCH_IN_PROGRESS';
-export const SEARCH_DONE = 'SEARCH_DONE';
```

Comment:
I find it too soon to extract constant for an App this size, even though I get it for bigger apps.